HoloRepositoryDemoApplication/Assets/Scripts/HologramDisplayScene.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using HoloStorageConnector;
+using TMPro;
+
+public class HologramDisplayScene : MonoBehaviour
+{   
+    [SerializeField]
+    private TextMeshProUGUI PatientName = null;
+    [SerializeField]
+    private TextMeshProUGUI PatientInfo = null;
+    [SerializeField]
+    private TextMeshProUGUI HologramTitle= null;
+    [SerializeField]
+    private TextMeshProUGUI HologramInfo = null;
+
+    public static Patient Patient;
+    public static Hologram Hologram;
+
+    void Start()
+    {
+        PatientName.text = $"{Patient.name.given} {Patient.name.family}";
+
+        PatientInfo.text = 
+            $"<b><size=15>Gender: </size></b>{Patient.gender}\n" +
+            $"<b><size=15>Date of birth: </size></b>{Patient.birthDate.Substring(0, 10)}";
+
+        HologramTitle.text = Hologram.title;
+
+        HologramInfo.text = 
+            $"<b><size=15>Description:</size></b>\n{Hologram.description}\n" +
```

Comment:
I think that the titles and the text should be the same size. Also, between the different blocks of "Title: Text..." could you add some padding? they are sitting a bit close on each other.
![image](https://user-images.githubusercontent.com/11090412/62793383-64921980-bac9-11e9-9492-26f0a3cec8f3.png)